playbooks/tasks/deploy_vm.yaml

```diff
@@ -24,18 +24,18 @@
 - block:
       - name: "Copy {{ item }} system disk on target"
         copy:
-          src: "{{ vm_file }}"
-          dest: "{{ vm_file_dest }}"
+          src: "{{ hostvars[item].vm_file | default(vm_file) }}"
+          dest: "{{ vm_file_dest | default ('/tmp/os_' + item + '.qcow2') }}"
```

Comment:
same here : 
`dest: "{{ qcow2tmpuploadfolder | default('/tmp') + '/os.qcow2' }}"`
and in all other mentions of the "vm_file_dest" variable